README.md

```diff
@@ -34,21 +34,21 @@ use rpm::signature::pgp::{Signer, Verifier};
 let raw_secret_key = std::fs::read("./test_assets/secret_key.asc")?;
 // It's recommended to use timestamp of last commit in your VCS
 let source_date = 1_600_000_000;
-let pkg = rpm::RPMBuilder::new("test", "1.0.0", "MIT", "x86_64", "some awesome package")
+let pkg = rpm::Builder::new("test", "1.0.0", "MIT", "x86_64", "some awesome package")
```

Comment:
I really don't like the lone `Builder` name.  I'm kind of conflicted on the others too, but this stands out as not being a clear name.

Likewise importing `rpm::Error` would be mutually exclusive with `dyn Error`.

My personal preference is to leave the prefix, but I'm fine with changing the casing i.e. `RpmError` and `RpmBuilder` and `RpmPackage`.

@drahnr @cmeister2 Opinions?

Reply:
I'd leave the prefix for `Package` and `Builder` at least

Reply:
>I really don't like the lone Builder name.

How about `PackageBuilder`?

>Likewise importing rpm::Error would be mutually exclusive with dyn Error.

In my experience, it's quite common for crates to define an `Error` type. For example, see the [`regex`](https://docs.rs/regex) crate. In some cases, crates even have per-module `Error` types, similarly to `std` which has a whole bunch of them.

>I'd leave the prefix for Package and Builder at least

Honestly, it's a bit weird to see `rpm::RpmPackage` and `rpm::RpmBuilder` in code.

Reply:
I have renamed `rpm::Builder` to `rpm::PackageBuilder`. I believe `rpm::Package` is a descriptive enough name.